internal/generator/vector/conf_test.go

```diff
@@ -411,9 +411,55 @@ if (.log_type == "audit"){
 ._id = encode_base64(uuid_v4())
 """
 
+[transforms.es_1_dedot_and_flatten]
+type = "lua"
+inputs = ["es_1_add_es_id"]
+version = "2"
+hooks.process = "process"
+source = """
+    function process(event, emit)
+        if event.log.kubernetes == nil then
+            return
+        end
+        dedot(event.log.kubernetes.pod_labels)
+        -- create "flat_labels" key
+        event.log.kubernetes.flat_labels = {}
```

Comment:
flat_labels in fluentd is an array, not a map.  is this the syntax here?

Reply:
yes, this is the syntax in `Lua`
this is example record captured after this change to `vector.toml`

```
            "flat_labels" : [
              "logging-infra=log-generator",
              "component=test",
              "pod-template-hash=78f8d8ddf8",
              "provider=openshift",
              "app_kubernetes_io/name=cluster-monitoring-operator",
              "app=cluster-monitoring-operator"
            ],
```